nomos-libp2p/Cargo.toml

```diff
@@ -43,6 +43,7 @@ tracing     = { workspace = true }
 zerocopy    = { version = "0.8.26", features = ["derive"] }
 
 [dev-dependencies]
+libp2p             = { workspace = true, features = ["plaintext", "yamux"] }
```

Comment:
Since we will merge soon a PR that removes default features, can we import this with `default-features` off and only enable the features we need? Probably turning them off should be enough.